app/lib/shared/Core/entities/DocumentEntity/mandatoryTest_6_1_27_6.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {any} doc
+ */
+export default function (doc) {
+  /** @type {Array<{ message: string; instancePath: string }>} */
+  const errors = []
+  let isValid = true
+
+  const checkedDocumentCategories = new Set(['security_advisory', 'vex'])
```

Comment:
@domachine: This should only test on `security_advisory` as `vex` has a special test for that:
```suggestion
  const checkedDocumentCategories = new Set(['security_advisory'])
```